opentreemap/api/tests.py

```diff
@@ -2083,7 +2083,7 @@ def test_json_requires_admin(self):
 class PasswordResetTest(OTMTestCase):
     def setUp(self):
         self.instance = setupTreemapEnv()
-        self.jim = User.objects.get(username="jim")
+        self.jim = User.objects.get(username="jim", password="password")
```

Comment:
Consider explaining in this PR conversation why we now need to also include `password` as a filter parameter.

Reply:
Actually, I think this is a mistake. Good catch!